calico-cloud/network-policy/hosts/kubernetes-nodes.mdx

```diff
@@ -72,6 +72,80 @@ ip-172-16-101-9.us-west-2.compute.internal-auto-hep     2021-05-12T22:16:47Z
 ip-172-16-102-63.us-west-2.compute.internal-auto-hep    2021-05-12T22:16:47Z
 ```
 
+### Use custom templates for host endpoints
+
+Template host endpoints are similar to automatic host endpoints, but with greater degree of customization. They are created automatically by the calico kube-controllers according to the `spec.controllers.node.hostEndpoint.templates`.
+You might want to consider using template host endpoints if you want automatic host endpoints to be created for certain Nodes only, or only for certain Node interfaces.
```

Comment:
Maybe this:

"Template HostEndpoints allow for tailored, automatic HostEndpoint generation. Defined in `spec.controllers.node.hostEndpoint.templates`, they enable creation based on specific Node or interface criteria, offering greater customization compared to default automatic HostEndpoints."